tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -159,3 +235,14 @@ Verify image manifest digest against vanilla docker
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  sha256:be3c11fdba7cfe299214e46edc642e09514dbb9bbefcd0d3836c05a1e0cd0642
+
+Attempt docker pull mitm
+    Get And Run MITMProxy Container
+    Get And Run Prepared Registry
+    ${ps}=  Run Docker Ps
+    ${mitm}=  Get Container Address  mitm  ${ps}
+    ${registry}=  Get Container Address  registry  ${ps}
+    ${vch2-IP}  ${vch2-params}=  Deploy Proxified VCH  ${registry}  ${mitm}
+    Pull And MITM Prepared Image  ${vch2-params}  ${registry}
+    Enable SSH on MITMed VCH
+    Check For Injected Binary  ${vch2-IP}
```

Comment:
It looks like you have leaked the second VCH, you need to clean that up.  Your Local Cleanup (not really a clear name) - can probably take care of that and make sure that you run a manual vic-machine delete --name VCH-XPLT

Reply:
Hmm, I was having some trouble getting it to remove w/ a keyword & Cleanup Dangling VCHes always catches it on the next test but I'll go ahead and do it manually.